src/tests/sqlsmith/src/runner.rs

```diff
@@ -0,0 +1,168 @@
+// Copyright 2022 Singularity Data
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use itertools::Itertools;
+use rand::{Rng, SeedableRng};
```

Comment:
Should this be using the rand crate from `madsim` instead https://docs.rs/madsim/latest/madsim/rand/index.html. That seems to be suggested by the documentation in the link.

Reply:
I remembered that @wangrunji0408 has eased this requirement someday, not sure 🤣

Reply:
IIRC, `getrandom` is the key, which will be replaced by Madsim.

Reply:
Yes, we can use the rand crate directly now! The documentation is outdated and should be updated. 🤣